bubble_sort.rb

```diff
@@ -0,0 +1,20 @@
+def bubble_sort(a)
+  (a.length - 1).times do
+    for i in 0...a.length - 1
```

Comment:
You can use the `..` operator instead (Two dots)

Reply:
I don't think I can because I don't want too iterate the last item.

Reply:
Aren't `(0...a.length - 1)` and `(0.. a.length)` the same? With the benefit that the second doesn't have an extra operation `(-)` and a magic number?

Reply:
No they are not, `0..a.length - 1` is same as `0...a.length`
But  `0..a.length` is not same as `0...a.length-1`